modules/bitgo/test/v2/unit/wallet.ts

```diff
@@ -2624,7 +2624,12 @@ describe('V2 Wallet:', function () {
       let signTxRequestForMessage;
       const messageSigningCoins = ['teth', 'tpolygon'];
       const messageRaw = 'test';
-      const expected: SignedMessage = { txRequestId: reqId.toString(), txHash, messageRaw, coin: 'teth' };
+      const expected: SignedMessage = {
+        txRequestId: reqId.toString(),
+        txHash,
+        messageRaw: `\u0019Ethereum Signed Message:\n${messageRaw.length}${messageRaw}`,
```

Comment:
Please add a comment to the code as to why we need a prefix here i.e a short summary of what is explained int the ticket. Otherwise it would look very sus